go.mod

```diff
@@ -4,6 +4,7 @@ go 1.22.1
 
 require (
 	github.com/BurntSushi/toml v1.4.0
+	github.com/bmatcuk/doublestar/v4 v4.6.1
```

Comment:
I propose using the latest available version v4.8.1 https://pkg.go.dev/github.com/bmatcuk/doublestar/v4

Reply:
done